src/components/charts/bar.js

```diff
@@ -17,6 +17,7 @@ import {
 
 import {
   modes,
+	modesDescription,
```

Comment:
Could you fix this irregular space and rename the variable to "tooltips"

Reply:
ok